### DISCUSSION TITLE
Not being able to RevalidatePath with Parallel Routing + Server Actions

### DISCUSSION THREAD

**Discussion by ditadi** [vercel/next.js#50895] [Help]
### Summary

Hey Guys, how are you?

So, recently I created a new repo with all new stuff on nextJS and other libraries and shared it on Twitter and it was seen a lot (10k views on Twitter).

Basically what my app does is:
- Authentication, Show a List of Posts and Create a new Post.
For all of this, I'm using Server Actions + revalidatePath + useOptimistic + useFormStatus.

But, now I want to edit a post with a Dialog Component using Parallel Routing (New). The Tricky point here is to Revalidate the List of Posts after the update. To Revalidate in the same URL is quite simple and it worked with the creation (which is in the same URL as the List - /dashboard), but when I do the Parallel Routing (/post/[id]/edit), didn't work well. He only shows the list updated when I reload manually.

This is the Repo: https://github.com/ditadi/web-boilerplate/tree/feat/update-post-parallel-routing

Did someone already try to glue RevalidatePath + Parallel Routing + Server Actions?

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:28 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6000
    Binaries:
      Node: 16.18.0
      npm: 8.19.2
      Yarn: 1.22.18
      pnpm: 8.3.1
    Relevant packages:
      next: 13.4.2
      eslint-config-next: 13.4.1
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.0.4

warn  - Latest canary version not detected, detected: "13.4.2", newest: "13.4.5-canary.7".
        Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before creating a new issue.
        Read more - https://nextjs.org/docs/messages/opening-an-issue
```

### Example

https://web-boilerplate-2-git-feat-update-post-parallel-routing-pulpo.vercel.app/dashboard

**Comment by pingustar** [comment]
I am interested in this as well
